pkg/ovirt/components/ClusterVms.jsx

```diff
@@ -115,9 +115,16 @@ const VmLastMessage = ({ vm }) => {
     if (!vm.lastMessage) {
         return null;
     }
-    const detail = (vm.lastMessageDetail && vm.lastMessageDetail.exception) ? vm.lastMessageDetail.exception: vm.lastMessage;
+
+    console.log('VmLastMessage(): vm.lastMessageDetail = ', vm.lastMessageDetail);
```

Comment:
Is console.log ok here? Or should we be using logDebug?

Reply:
Good catch, thanks.
Leftover from debugging.